pkg/appmanager/resourceConfig.go

```diff
@@ -2209,8 +2211,7 @@ func (rc *ResourceConfig) RemoveMonitor(pool, monitor string) bool {
 		}
 	}
 	for i, mon := range rc.Monitors {
-		name := strings.Split(monitor, "/")[2]
-		if mon.Name == name {
+		if strings.Contains(monitor, mon.Name) {
```

Comment:
Why was this needed for the change?

Reply:
If a monitor doesn't exist for a pool, then the "monitor" string won't get set. The array index would crash. I could make this safer by wrapping this for loop in a conditional; whether monitor is empty or not.